### PR TITLE
Update/0.12.2

### PR DESCRIPTION
**Pull request by Dominion5254**
* Update upstream to 0.12.2 - Full release notes available [here](https://github.com/lnbits/lnbits/releases/tag/0.12.2).
  * Create and save default Superuser username and password in Properties
  * Add action to reset Superuser password
  * Default username-password as sole Auth method for fresh installations
  * Update instructions

### CODE REVIEW

File: docker_entrypoint.sh [hunk 1]
```diff
@@ -161,6 +176,58 @@ fi
 poetry run lnbits --port $LNBITS_PORT --host $LNBITS_HOST &
 lnbits_process=$!
 
+while ! [ -f $FILE ]; do {
```
Comment (gStart9) [inline docker_entrypoint.sh]:
```suggestion
while ! [ -f $FILE ]; do
```

File: docker_entrypoint.sh [hunk 2]
```diff
@@ -161,6 +176,58 @@ fi
...
+  echo "Waiting for DB to be created..."
+  sleep 10
+}; done
```
Comment (gStart9) [inline docker_entrypoint.sh]:
```suggestion
done
```

File: docker_entrypoint.sh [hunk 3]
```diff
@@ -161,6 +176,58 @@ fi
...
+
+while true; do {
```
Comment (gStart9) [inline docker_entrypoint.sh]:
```suggestion
while true; do
```

File: docker_entrypoint.sh [hunk 4]
```diff
@@ -161,6 +176,58 @@ fi
...
+  ACCOUNTS_COLUMNS=$(sqlite3 ./data/database.sqlite3 'PRAGMA table_info(accounts);')
+  if echo "$ACCOUNTS_COLUMNS" | grep -q "username"; then
+    break
+  fi
+  echo "Waiting for migrations to complete..."
+  sleep 10
+}; done
```
Comment (gStart9) [inline docker_entrypoint.sh]:
```suggestion
done
```

**Review by gStart9 — COMMENTED** [review]
I haven't tried to run this yet but it looks good on cursory inspection (after the removal of those curly braces).

**Review by gStart9 — COMMENTED** [review]
Tested and working!

File: docker_entrypoint.sh (hunk 1 above)
Comment (Dominion5254) [reply docker_entrypoint.sh]: Removed

File: docker_entrypoint.sh (hunk 2 above)
Comment (Dominion5254) [reply docker_entrypoint.sh]: Removed

File: docker_entrypoint.sh (hunk 3 above)
Comment (Dominion5254) [reply docker_entrypoint.sh]: Removed

File: docker_entrypoint.sh (hunk 4 above)
Comment (Dominion5254) [reply docker_entrypoint.sh]: Removed

**Review by gStart9 — APPROVED** [review]

**Pull request merged**